airgun/views/task.py

```diff
@@ -1,6 +1,7 @@
 from wait_for import wait_for
 from widgetastic.widget import Table, Text, View
 from widgetastic_patternfly import BreadCrumb
+from widgetastic_patternfly4 import Pagination
```

Comment:
Thanks for adding this back, adding suggestion just for consistency
```suggestion
from widgetastic_patternfly4 import Pagination as PF4Pagination
```